src/mlpack/core/math/shuffle_data.hpp

```diff
@@ -46,11 +47,13 @@ void ShuffleData(const MatType& inputPoints,
                  const LabelsType& inputLabels,
                  MatType& outputPoints,
                  LabelsType& outputLabels,
-                 const std::enable_if_t<arma::is_SpMat<MatType>::value>* = 0)
+                 const std::enable_if_t<arma::is_SpMat<MatType>::value>* = 0,
+                 const std::enable_if_t<!arma::is_Cube<MatType>::value>* = 0)
 {
   // Generate ordering.
   arma::uvec ordering = arma::shuffle(arma::linspace<arma::uvec>(0,
       inputPoints.n_cols - 1, inputPoints.n_cols));
+//  std::cout << "ordering:\n" << ordering.t();
```

Comment:
Looks like we can remove the debug message here.

Reply:
Oops, fixed.